torch/testing/_internal/opinfo/definitions/linalg.py

```diff
@@ -1517,8 +1517,7 @@ def make_input():
         "linalg.lstsq",
         aten_name="linalg_lstsq",
         variant_test_name="grad_oriented",
-        # gradchecks for forward AD fails with multi-Tensor outputs
-        op=lambda a, b, driver: torch.linalg.lstsq(a, b, driver=driver)[0],
+        op=lambda a, b, driver: torch.linalg.lstsq(a, b, driver=driver),
```

Comment:
nit: If the tests are green, we can remove this line altogether.

Reply:
I incorrectly thought the only relevant tests were in `test/test_ops.py`. This actually still doesn't work for `test/test_ops_fwd_gradients.py`, but `op=lambda a, b, driver: torch.linalg.lstsq(a, b, driver=driver)[:2]` works on my machine, so I just changed to that.

Reply:
I did rerun the workflow.  Let's see how it goes now.